### DISCUSSION TITLE
Request for Camera and NVR Brand Setup Tutorials in Viseron Documentation

### DISCUSSION THREAD

**Discussion by hayschan** [roflcoopter/viseron#769] [General]
Hello Viseron Community,

The existing documentation is comprehensive for initial setup and configuration. However, integrating specific camera and NVR (Network Video Recorder) brands could use more detailed tutorials. Many users employ devices from popular brands, and having precise, model-specific setup instructions would be invaluable. It will act as a starting point for newcomers.

I propose we add:
- Camera Setup Guides: Step-by-step tutorials for popular camera brands, tailored to specific models.
- NVR Integration Guides: Detailed tutorials for integrating major NVR brands with Viseron, addressing common configurations and troubleshooting.

Such detailed guides would ease the setup process, particularly for new users, and enhance the overall user experience by providing practical, replicable examples.

I appreciate any feedback or contributions towards developing these resources and enhancing our community's capabilities!

**Comment by roflcoopter** [comment]
That's a great idea! What brands do you have at hand?
I only own a few Hikvision cameras so I don't have that many devices to test with sadly

**Reply by hayschan** [reply]
I am using Hikvision NVR (I have to check the exact model) and Hikvision cameras. 

I am a full-time software developer so I am quite familiar with the environment setup and things. But this is my first on NVR and cameras so I am a bit hesitant. I will need some help from you along the way.

If I can make it through, I will also PR a guide for people in similar situation to follow in the future.

**Reply by roflcoopter** [reply]
Sounds great! I'll gladly answer any questions.

PRs are always welcome, right now I am pretty much the only contributor which makes progress a little slow at times
